docs/0.4/examples/gameroom/walkthrough/index.md

```diff
@@ -1280,14 +1345,14 @@ At the end of the operation, the `gameroom_service` table looks like this:
     <th><code>arguments</code></th>
   </tr>
   <tr>
-    <td><code>gameroom_acme-node-000</code></td>
+    <td><code>01234-ABCDE::gr01</code></td>
```

Comment:
i think this should only be gr01 (no circuit id)

Reply:
This is in the `service_id` column, so shouldn't it be the full service ID?